java/ql/src/change-notes/2025-03-03-fix-improper-intent-verification-query.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+* Fixed false positive in CWE-925 by requiring the `onReceive` method must be non-empty
```

Comment:
```suggestion
* Overrides of `BroadcastReceiver::onReceive` with no statements in their body are no longer considered unverified by the `java/improper-intent-verification` query. This will reduce false positives from `onReceive` methods which do not perform any actions.
```